modules/reactor/include/meshgenerators/ReactorGeometryMeshBuilderBase.h

```diff
@@ -12,6 +12,49 @@
 #include "MeshGenerator.h"
 #include "libmesh/elem.h"
 
+namespace RGMB
+{
+
+// General global quantities for mesh building
+static const std::string mesh_dimensions = "mesh_dimensions";
```

Comment:
@loganharbour wanted constexpr. does it work?

Reply:
constexpr does not work with std::string

https://stackoverflow.com/questions/27123306/is-it-possible-to-use-stdstring-in-a-constexpr